sleap/gui/shortcuts.py

```diff
@@ -70,6 +70,10 @@ def __init__(self):
         self._shortcuts = self._process_shortcut_dict(shortcuts)
         self._defaults = self._process_shortcut_dict(defaults)
 
+        # TODO: Add switch view hotkey
+        self._shortcuts["goto next view"] = QKeySequence.fromString("V")
+        self._shortcuts["goto prev view"] = QKeySequence.fromString("Shift+V")
```

Comment:
Doing things this way will override the user-set shortcut to always be our default shortcut. The only file we want to make changes to is `shortcuts.yaml`. Checkout #1444 that does something similar.